Assets/BossRoom/Scripts/Game/ConnectionManagement/ClientGameNetPortal.cs

```diff
@@ -156,6 +156,7 @@ private void OnDisconnectOrTimeout(ulong clientID)
                     case ConnectStatus.UserRequestedDisconnect:
                     case ConnectStatus.HostEndedSession:
                     case ConnectStatus.ServerFull:
+                    case ConnectStatus.IncompatibleBuildType:
```

Comment:
Shouldn't we also have some feedback to players? If it just disconnects with no message, we wouldn't know why?

Reply:
The message is already there. It was just not being sent because it was throwing an exception here